graylog2-server/src/main/java/org/graylog2/rest/resources/system/GrokResource.java

```diff
@@ -111,10 +121,10 @@ public Response bulkUpdatePatterns(@ApiParam(name = "patterns", required = true)
         }
 
         grokPatternService.saveAll(GrokPatterns.fromSummarySet(patternList.patterns()), replace);
-
+        clusterBus.post(GrokPatternsChangedEvent.create(Collections.emptySet(), updatedPatternNames));
```

Comment:
correct, this is misleading as the reloader actually reloads everything. I'll put the new pattern names in to avoid confusion down the road